### ISSUE TITLE
Presentation mode should not allow editing

### ISSUE THREAD

**Issue by TheItachiUchiha** [Allminds/mind-it#5]
Ideally, presentation mode should be a read only interface. Currently, mind-it allows us to edit even when we are in presentation mode.

**Comment by mahadevTW** [comment]
hello  
@TheItachiUchiha 
We are working on it, you will get updated code soon.

**Comment by SwapnilBGaikwad** [comment]
@TheItachiUchiha we have presentation currently with editing and also with the collaboration and because of  dynamic updates we have problem in the navigation. So for now if you are using presentation mode keep your mindmap private. 

For this release we are working on it and if possible we will do it in this release itself or you have have to wait for the next release.

Thank you.

**Comment by chiragsdoshi** [comment]
I disagree. I think its great that presentation mode allows for edit and all other actions on mindmaps. I use it as a full screen mode while editing as well.
